vanilla-framework/vanilla-framework
# Update docs/tables page content

## Done
- Updated copy and merged all sections into one 'Data tables' section 👍 
- Updated demo example to showcase live example 😉 
- Check content matches copy doc [here](https://docs.google.com/document/d/1jhKcD6iKKUdmCM_6Gh-DZu30QfxLRz6Mr0x9O_WfrpA/edit#)

## QA
- Pull code
- Run /docs > ./run
  - Check http://0.0.0.0:8104/en/base/table
- Pull code
- Run ./run
- Review updated examples
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/table/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/tables/table-sortable/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/tables/table-expanding/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/tables/table-mobile-card/

## Details
- Fixes https://github.com/ubuntudesign/vanilla-design/issues/350

## Screenshots
![Screenshot 2019-03-20 at 11 08 57](https://user-images.githubusercontent.com/17748020/54680086-de1c7180-4b00-11e9-8272-3837949718f4.png)
![Screenshot 2019-03-20 at 11 10 38](https://user-images.githubusercontent.com/17748020/54680087-de1c7180-4b00-11e9-80ff-434f19f38005.png)
